calendar-cli.py

```diff
@@ -11,8 +11,8 @@
 from datetime import datetime, timedelta, date
 import dateutil.parser
 from dateutil.rrule import rrulestr
-from icalendar import Calendar,Event,Todo,Journal
-import caldav
+from icalendar import Calendar,Event,Todo,Journal,Alarm
+#import caldav
```

Comment:
How can it work without the caldav import?

The dev-version of calendar-cli may break because it depends on the dev-version of the caldav library.

I really should get my acts together and get new versions of caldav and calendar-cli released soon ... though I'm drowning a bit in my backlog at the moment ... :/